### DISCUSSION TITLE
Document outlines

### DISCUSSION THREAD

**Discussion by sarah-tonin** [knowledgefutures/pubpub#2511] [Ideas]
Enhancement requests are requests to make changes to an existing PubPub feature. If you'd like to suggest a new feature, please post in the ideas section of our [https://github.com/pubpub/pubpub/discussions](discussion forum).

_Before filing your enhancement, please search existing issues to make sure a similar one doesn't already exist. If one does, please leave a comment or reaction in support of the issue rather than creating a new one._

_To fill out the request, please leave the below headers (**) in tact, and fill out each section. If a section isn't applicable, leave it blank or write n/a under it. Please do not delete any template sections._

### What's your enhancement request?
On really long pubs, especially ones with lots of (sub)headers, it would be useful to have a clickable/collapse-able outline along the side (perhaps on the right). I'm thinking kind of like Google Docs has. 

### What would this enhancement help you accomplish?
This would help jump to different sections of a document which would be useful both as an editor and a writer. Perhaps too even as a reader to know what the different headers are and thus what the pub will be about/cover. 

---

**Comment by gabestein** [comment]
Converting to a discussion to gauge interest, since this one is fairly speculative.
